assets/js/hooks/animation.js

```diff
@@ -1,5 +1,9 @@
 export default {
 
+    hiddenClasses() {
```

Comment:
How tailwind/purge generates those classes, if they are not in existing codebase already or will be removed?